mne/io/reference.py

```diff
@@ -181,6 +185,22 @@ def add_reference_channels(inst, ref_channels, copy=True):
         raise TypeError("inst should be Raw, Epochs, or Evoked instead of %s."
                         % type(inst))
     nchan = len(inst.info['ch_names'])
+
+    # "zeroth" EEG electrode dig points is reference
+    ref_dig_loc = [dl for dl in inst.info['dig'] if (
+                   dl['kind'] == FIFF.FIFFV_POINT_EEG and
+                   dl['ident'] == 0)]
+    if len(ref_channels) > 1 or len(ref_dig_loc) != len(ref_channels):
+        ref_dig_array = np.zeros(12)
+        warn('The locations of multiple reference channels are ignored '
+             '(set to zero).')
+    else:  # n_ref_channels == 1 and a single ref digitization exists
+        ref_dig_array = np.concatenate((ref_dig_loc[0]['r'],
+                                       ref_dig_loc[0]['r'], np.zeros(6)))
+        # Replace the (possibly new) Ref location for each channel
+        for idx in pick_types(inst.info, meg=False, eeg=True):
```

Comment:
I don't *think* so, in Neuromag at least they are bipolar. But someone should probably check.

Also this should have `exclude=()` so that it gets set even for bad EEG channels.